### PR TITLE
Release 0.41.2

### PR DESCRIPTION
**Pull request by lfittl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
 # Changelog
 
+## 0.41.2      2021-11-18
+
+* Add two additional log_line_prefix settings
+  - `%p-%s-%c-%l-%h-%u-%d-%m `
+  - `%m [%p][%b][%v][%x] %q[user=%u,db=%d,app=%a] `
+* Change schema collection message to warning during test run
+  - This helps discover schema collection issues, e.g. due
+    to connection restrictions or pother ermission problems
```
Comment (seanlinsley) [inline CHANGELOG.md:10]: typo here: `pother ermission`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+* Add support for multiple domain constraints
```
Comment (uhoh-itsmaciek) [inline CHANGELOG.md:11]: This is technically "fix issue with" since multiple constraints would break schema collection entirely before, but NBD.

Comment (lfittl) [reply CHANGELOG.md]: Good point, adjusted

File: CHANGELOG.md (hunk 1 above)
Comment (lfittl) [reply CHANGELOG.md]: fixed!

**Review by seanlinsley — APPROVED** [review]

**Review by uhoh-itsmaciek — APPROVED** [review]

**Pull request merged**